ui/webui/Makefile.am

```diff
@@ -71,7 +71,7 @@ COCKPIT_REPO_FILES = \
 	$(NULL)
 
 COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
-COCKPIT_REPO_COMMIT = 54a8ebd2dfe4d53bcfa5db660701469e3f21dbba # 288.2 + cockpit-po-plugin fix
+COCKPIT_REPO_COMMIT = 84c7ee507b57485b5edd97ed3321982a59032f20 # 289 + run-tests machine_class fix
```

Comment:
Just a random though, unrelated to the PR itself - perhaps this could be a separate file, same as `branch-config.mk` ?